openapi/components/schemas/Coupon/CouponRestrictions/restrict-to-bxgy.yaml

```
@@ -0,0 +1,45 @@
+description: restrict-to-bxgy restrictions.
+allOf:
+  - $ref: ../CouponRestriction.yaml
+  - $ref: ../RedemptionRestriction.yaml
+  - type: object
+    required:
+      - buy
+      - get
+    properties:
+      buy:
+        type: array
+        description: One or more plan and quantity definitions that must be purchased for receiving all plans and quantity defined in `get`.
```

nitpick: sentence is difficult to follow
```suggestion
        description: One or more plan and quantity definitions that must be purchased to receive all plans and quantities that are defined in `get`.
```